### ISSUE TITLE
Form fields: define common attributes as props

### ISSUE THREAD

**Issue by LeBenLeBen** [liip/chusho#166]
The following components:

- CCheckbox
- CRadio
- CTextarea
- CTextField

Should define the following props:

- readonly (CTextarea & CTextField only)
- disabled
- required

So they are passed to the config class methods and allows to customize the element style based on them.

**Comment by ithiel883** [comment]
can i get started on this issue?

**Comment by LeBenLeBen** [comment]
@ithiel883 It’s already done actually, I’m doing it in parallel of #149. Thanks for asking!
